src/apple/config/mod.rs

```diff
@@ -136,6 +144,12 @@ impl Error {
                 msg,
                 format!("`{}.project-dir` invalid: {}", super::NAME, err),
             ),
+            Self::IosVersionDoubleInvalid(err) => {
+                Report::error(msg, format!("iOS version double is not valid: {}", err))
```

Comment:
These reports should follow the precedent of ``"`{}.field-name` invalid: {}"`` from other variants